src/sentry/migrations/0235_add_metrics_indexer_table.py

```diff
@@ -0,0 +1,56 @@
+# Generated by Django 2.2.24 on 2021-09-30 17:23
+
+from django.db import migrations, models
+
+import sentry.db.models.fields.bounded
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn't be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = False
+
+    # This flag is used to decide whether to run this migration in a transaction or not.
+    # By default we prefer to run in a transaction, but for migrations where you want
+    # to `CREATE INDEX CONCURRENTLY` this needs to be set to False. Typically you'll
+    # want to create an index concurrently when adding one to an existing table.
+    # You'll also usually want to set this to `False` if you're writing a data
+    # migration, since we don't want the entire migration to run in one long-running
+    # transaction.
+    atomic = True
+
+    dependencies = [
+        ("sentry", "0234_grouphistory"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="MetricsKeyIndexer",
+            fields=[
+                (
+                    "id",
+                    sentry.db.models.fields.bounded.BoundedBigAutoField(
+                        primary_key=True, serialize=False
+                    ),
+                ),
+                ("organization_id", sentry.db.models.fields.bounded.BoundedBigIntegerField()),
+                ("string", models.CharField(max_length=200)),
+            ],
+            options={
+                "db_table": "sentry_metricskeyindexer",
+            },
+        ),
+        migrations.AddConstraint(
+            model_name="metricskeyindexer",
+            constraint=models.UniqueConstraint(
+                fields=("organization_id", "string"), name="unique_org_string"
+            ),
```

Comment:
Does this also add the index ?

Reply:
yeah, and also the auto increment happens to the `id` because the default is set to the `nextval()` for the sequence

```
sentry=# \d sentry_metricskeyindexer
                                     Table "public.sentry_metricskeyindexer"
     Column      |          Type          |                               Modifiers
-----------------+------------------------+-----------------------------------------------------------------------
 id              | bigint                 | not null default nextval('sentry_metricskeyindexer_id_seq'::regclass)
 organization_id | bigint                 | not null
 string          | character varying(200) | not null
Indexes:
    "sentry_metricskeyindexer_pkey" PRIMARY KEY, btree (id)
    "unique_org_string" UNIQUE CONSTRAINT, btree (organization_id, string)
```